### DISCUSSION TITLE
how can I install oh-my-posh in git bash windows terminal

### DISCUSSION THREAD

**Discussion by mohdh34m** [JanDeDobbeleer/oh-my-posh#2718] [Troubleshoot] [answered]
I have installed it in `PowerShell and it worked`, how can I install `oh-my-posh in git bash`?
I Have installed oh-my-posh with `scoop` then I added this code to `.bashrc`:

`eval "$(oh-my-posh --init --shell bash --config ~/amro.omp.json)"`

but unfortunately didn't work

**Comment by JanDeDobbeleer** [comment, marked as answer]
That should work unless if the oh-my-posh executable isn't in your PATH. What error do you get?

**Reply by mohdh34m** [reply]
there's no error,

[![Screenshot-2022-08-28-193148.png](https://i.postimg.cc/YSdCDJNQ/Screenshot-2022-08-28-193148.png)](https://postimg.cc/f30sJqrb)

oh-my-posh PATH:
`"C:\Users\myuser\scoop\apps\oh-my-posh"`

Themes PATH:
`"C:\Users\myuser\AppData\Local\oh-my-posh\themes"`

.bashrc PATH:
`"C:\Users\myuser\.bashrc"`

**Reply by JanDeDobbeleer** [reply]
@mohdh34m in that case, the `.bashrc` isn't loaded. Usually the `.profile` has something line this at the end:

```bash
if [ -f ~/.bashrc ]; then
    . ~/.bashrc
fi
```

**Reply by mohdh34m** [reply]
There are 2 errors

[![Screenshot-2022-08-28-200430.png](https://i.postimg.cc/NFvTMVN9/Screenshot-2022-08-28-200430.png)](https://postimg.cc/5YpjKPWb)

**Reply by JanDeDobbeleer** [reply]
@mohdh34m the first one isn't an oh-my-posh error, the second isn't an error but is an issue with the font not being set in your terminal and mixing Windows and bash paths (that's why the config doesn't load).

**Reply by mohdh34m** [reply]
Ok Thank you, Solved
